scripts/oes-data-migration-scripts/migration_v3.12.x_to_v4.0.x.py

```diff
@@ -1458,21 +1486,22 @@ def formPipelineJson(pipelineId, dbPipelineJson, stageJson):
         raise e
 
 
-
 def postingGateJsonSQL(pipelineJson):
     try:
         pipeline_id = pipelineJson['id']
-        update_data = json.dumps(pipelineJson), isd_admin_username, pipeline_id	
-        mysqlcursor_front50 = spindb_front50.cursor()
+        update_data = json.dumps(pipelineJson), isd_admin_username, pipeline_id        
         sql = "UPDATE pipelines SET body = %s, last_modified_by = %s WHERE id = %s"        
-        mysqlcursor_front50.execute(sql, update_data)
+        if spin_db_type == 'sql':
+           mysqlcursor_front50 = spindb_front50_sql.cursor()
+           mysqlcursor_front50.execute(sql, update_data)
```

Comment:
shouldn't we close cursor ? any issue if we close cursor here ?

Reply:
added to close cursor